### ISSUE TITLE
Checking a query executed succesfully

### ISSUE THREAD

**Issue by jjokham** [paragonie/easydb#104]
how can you verify if a query was executed successfully. is there any bool returned?

**Comment by colshrapnel** [comment]
You don't need that. This is not how the code is written in 2019.
You don't check for the query result but instead just assume it was a success.
That's all.

There are many outdated tutorials that tell you to check the query results but they are plain wrong, just ditch them away.

In a rare case when you need to handle an error, you can catch the exception, but most of time you shouldn't do it.
So in a nutshell just run your query and tell a user it was a success. that's all.

**Comment by paragonie-scott** [comment]
@colshrapnel is correct here.

The alternative answer is to do this:

```php
class Foo
{
    /** @var EasyDB $db */
    private $db;

    /**
     * @return bool
     * @throws PDOException
     */
    function bar(): bool
    {
        $this->db->beginTransaction();
        
        // Do your queries here...
        $this->db->query(/* bar */);
        
        return $this->db->commit();
    }
}
```

**Comment by jjokham** [comment]
Thank you @colshrapnel and @paragonie-scott
